cypress/e2e/common/cmsSteps.js

```diff
@@ -547,4 +563,30 @@ And('user selects {string} from Source View dropdown', (sourceDropdown) => {
 
 And('user enters {string} in Raw HTML Content under Secondary Contents', (value) => {
     cy.get('table[id*="field-secondary-contents"]').find('textarea[name*="field_raw_html"]').type(value);
+})
+
+When('user clicks on {string} option button',(deleteBtn)=>{
+cy.get("[id*='edit-delete']").click();
+});
+When('user confirms {string} action',(deleteConfirm)=>{
+    if(deleteConfirm.includes('Delete')){
+        cy.get(`div[class*='ui-dialog-buttonset form-actions'] button:contains('${deleteConfirm}')`).click();
+    }else {
+        cy.get(`input[value='${deleteConfirm}']`).click()
+    }
+
+});
```

Comment:
Is this only deletions?  Or can it be used for other confirmations?  I _think_ this is aimed at addressing English versus Spanish, in which case `deleteConfirm` should be something more descriptive such as `buttonText`.

The hard-coded check for `'Delete'` leaves me wondering whether the condition is needed or if there should just be a single button selection.

Reply:
addressed ( added a comment to explain why)